lib/plugins/aws/index.js

```diff
@@ -101,6 +101,14 @@ class SDK {
       return found.PhysicalResourceId;
     });
   }
+
+  getAccountId(iamRoleArnLambda) {
```

Comment:
Is this still necessary? We've recently merged a PR which removed the iamRoleArnLambda.